packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-scheme-type.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  ComponentsElement,
+  OpenApi3_1Element,
+  SecurityRequirementElement,
+  SecuritySchemeElement,
+  isSecuritySchemeElement,
+  isComponentsElement,
+} from '@swagger-api/apidom-ns-openapi-3-1';
+import {
+  AnnotationElement,
+  toValue,
+  cloneShallow,
+  isObjectElement,
+} from '@swagger-api/apidom-core';
+
+type SecuritySchemeTypePluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const securitySchemeTypeRefractorPlugin =
+  ({ annotations }: SecuritySchemeTypePluginOptions) =>
+  () => {
+    const removedSecuritySchemes: SecuritySchemeElement[] = [];
+    const annotation = new AnnotationElement(
+      '"mutualTLS" type is not supported in OpenAPI 3.0.3. It will be removed from the converted document.',
+      { classes: ['error'] },
+      { code: 'mutualTLS' },
+    );
+
+    return {
+      visitor: {
+        OpenApi3_1Element(element: OpenApi3_1Element) {
+          if (!isComponentsElement(element.components)) return undefined;
+          if (!isObjectElement(element.components.securitySchemes)) return undefined;
+
+          element.components.securitySchemes.forEach((value) => {
+            if (isSecuritySchemeElement(value) && toValue(value.type) === 'mutualTLS') {
+              removedSecuritySchemes.push(value);
+            }
+          });
+
+          return undefined;
+        },
+        ComponentsElement(element: ComponentsElement) {
```

Comment:
In light of https://github.com/swagger-api/apidom/pull/3812 here is my suggestion for `ComponentsElement`:

```js
ComponentsElement(element: ComponentsElement) {
  if (!isObjectElement(element.securitySchemes)) return undefined;

  element.securitySchemes.forEach((value, key) => {
    if (isSecuritySchemeElement(value) && toValue(value.type) === 'mutualTLS') {
      if (!removedSecuritySchemes.includes(value)) removedSecuritySchemes.push(value);
      element.remove(toValue(key));
    }
  });
}
```